tests/functional/func_test.py

```diff
@@ -53,7 +53,7 @@ def seed_rnd():
         import base64
         import socket
         import ssl
-        rnd = base64.decodestring("""k/zFvqjGWdhStmhfOeTNtTs89P8soknF1J9kSQrz8hKdrjIutqTXMfIqCNUb7DXrMykMW+wKd1Pg
+        rnd = base64.decodebytes("""k/zFvqjGWdhStmhfOeTNtTs89P8soknF1J9kSQrz8hKdrjIutqTXMfIqCNUb7DXrMykMW+wKd1Pg
```

Comment:
`decodebytes` is not available in Python 2, but we have CI jobs for Python 2 builds/functests.